{
	"schema_version": "1.6.0",
	"id": "RHSA-2024:1930",
	"modified": "2024-11-24T15:11:58Z",
	"published": "2024-09-16T18:44:03Z",
	"related": [
		"CVE-2024-29156"
	],
	"summary": "Red Hat Security Advisory: Red Hat OpenStack Platform 17.1 (openstack-tripleo-heat-templates and python-yaql) security update",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.1/AV:L/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "Red Hat:openstack:17.1::el8",
				"name": "openstack-tripleo-heat-templates",
				"purl": "pkg:rpm/redhat/openstack-tripleo-heat-templates"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:14.3.1-17.1.20231103003748.2.el8ost"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2024:1930.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:openstack:17.1::el8",
				"name": "python-yaql",
				"purl": "pkg:rpm/redhat/python-yaql"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:1.1.3-11.el8ost"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2024:1930.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:openstack:17.1::el8",
				"name": "python3-yaql",
				"purl": "pkg:rpm/redhat/python3-yaql"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:1.1.3-11.el8ost"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2024:1930.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/errata/RHSA-2024:1930"
		},
		{
			"type": "ARTICLE",
			"url": "https://access.redhat.com/security/updates/classification/#important"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2269112"
		},
		{
			"type": "ADVISORY",
			"url": "https://security.access.redhat.com/data/csaf/v2/advisories/2024/rhsa-2024_1930.json"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2024-29156"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2024-29156"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2024-29156"
		},
		{
			"type": "ARTICLE",
			"url": "https://bugs.launchpad.net/murano/+bug/2048114"
		}
	]
}
